File: C_Programming/Competative/Assignments/Assignment_3/program5.c
//Accept one charachter from user and check whether that character is vowel (a,e,i,o,u) or not
//Input : E   Output : TRUE
//Input : D   Output : FALSE

#include<stdio.h>

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL CheckVowel(char cValue)
{
    if(cValue == 'A' || 'E' ||'I' || 'O' || 'U')
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    char CValue = '\0';
    BOOL bChar = FALSE;

    printf("Enter the character\n");
    scanf("%c", &CValue);

    bChar = CheckVowel(CValue);

    if(bChar == TRUE)
    {
        printf("It is vowel\n");
    }
    else
    {
        printf("It is not Vowel\n");
    }


    return 0;
}

File: Java_Programming/Competative/Assignments/Assignment_3/program5.java
package Competative.Assignments.Assignment_3;

//Accept one charachter from user and check whether that character is vowel (a,e,i,o,u) or not
//Input : E   Output : TRUE
//Input : D   Output : FALSE

import java.util.*;

class DisplayX
{
    public boolean CheckVowel(char cValue)
    {
        if(cValue == 'A' || cValue =='E'|| cValue =='I' || cValue == 'O' || cValue == 'U')
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program5 
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        char CValue = '\0';
        boolean bChar = false;

        System.out.println("Enter the character : ");
        CValue = sobj.next().charAt(0);

        DisplayX dobj = new DisplayX();

        bChar = dobj.CheckVowel(CValue);

        if(bChar == true)
        {
            System.out.println("It is vowel ");
        }
        else
        {
            System.out.println("It is not Vowel ");
        }

    }
}
